frontend/public/components/alert.tsx

```diff
@@ -385,13 +385,9 @@ const AlertsPage_ = connect(listStateToProps)(class InnerAlertsPage_ extends Rea
 
     return <React.Fragment>
       <NavTitle title="Monitoring Alerts" />
-      <div className="co-m-pane__filter-bar">
-        <div className="co-m-pane__filter-bar-group">
```

Comment:
So that this filter bar stays consistent with the others, I recommend:

```
<div className="co-m-pane__filter-bar-group">
  <AlertsPageDescription />
</div>
 <div className="co-m-pane__filter-bar-group co-m-pane__filter-bar-group--filter">	
   <TextFilter defaultValue={this.defaultNameFilter} label="Alerts by name" onChange={this.applyTextFilter} />	
 </div>
```
And then no changes to `.monitoring-heading`.

Note:  recommended changes are untested, but should work.

Reply:
Thanks @rhamilto. I updated it to use your approach.